components/button/button_test.go

```diff
@@ -0,0 +1,33 @@
+package button_test
```

Comment:
Move to fake, it's testing that package, the helpers in the interface package are tested in the resource graph.